pkgs/tools/misc/turbo/wrapper.nix

```
@@ -0,0 +1,9 @@
+{ lib, runCommand, makeWrapper, turbo-unwrapped
+, disableTelemetry ? true, disableUpdateNotifier ? true }:
+
+runCommand "turbo" { nativeBuildInputs = [ makeWrapper ]; } ''
```

I assume `runCommand` supports these inputs.

```suggestion
runCommandNoCC "turbo-${version}" { 
  inherit (turbo-unwrapped) version meta; 
  pname = "turbo";
  nativeBuildInputs = [ makeWrapper ]; 
} ''
```